src/components/embed.ts

```diff
@@ -1,229 +1 @@
-import {JSONAble} from '../interfaces';
```

Comment:
Looked like this was kind of unfinished @alii - let me know if I'm wrong though! Removed it for now to try and simplify things